src/main/resources/static/js/payment/checkPayment.js

```diff
@@ -0,0 +1,58 @@
+$(document).ready(
+    function(){
+        setState();
+        addListeners();
+    }
+);
+
+function setState() {
+    var btnSave = $('#btnSave');
+    var inputAmount = $('#inputAmount');
+
+    var amount = parseFloat(inputAmount.val());
+    var due = parseFloat($('#amountDue').val());
+
+    if (isNaN(amount) || isNaN(due)) {
+        btnSave.attr("disabled", true);
+        return;
+    }
+
+    if (amount <= 0) {
+        inputAmount.val("").select();
+        btnSave.attr("disabled", true);
+        alert("Payments must be a positive number");
+        return;
+    }
+
+    // Make sure auth number is filled in if the field exists (cash and check payment types)
+    var inputAuth = document.getElementById('inputCheckAuth');
+
+    if (inputAuth == null) {
+        inputAuth =  document.getElementById('inputAuthNumber');
+    }
```

Comment:
With the HTML files broken up, this file doesn't need to worry about #inputAuthNumber any more (since that's on the card page)

Reply:
changed for checkPayment and cardPayment